diff --git a/node_modules/vite/dist/node/chunks/config.js b/node_modules/vite/dist/node/chunks/config.js
index 2b39a20..d9ff548 100644
--- a/node_modules/vite/dist/node/chunks/config.js
+++ b/node_modules/vite/dist/node/chunks/config.js
@@ -2227,7 +2227,7 @@ function optimizeSafeRealPathSync() {
 			return;
 		}
 	}
-	exec("net use", (error$1, stdout) => {
+	exec("net use", { windowsHide: true }, (error$1, stdout) => {
 		if (error$1) return;
 		const lines = stdout.split("\n");
 		for (const line of lines) {
